solidity/test/BondedECDSAKeepFactoryTest.js

```diff
@@ -1367,6 +1368,27 @@ contract("BondedECDSAKeepFactory", async (accounts) => {
     })
   })
 
+  describe("hasAuthorization", async () => {
+    before(async () => {
+      await initializeNewFactory()
+      await initializeMemberCandidates()
+    })
+
+    it("returns true if operator authorized the factory", async () => {
```

Comment:
Technically it's not the operator who authorizes the factory.

```suggestion
    it("returns true if operator is authorized for the factory", async () => {
```